frontend/src/redux/roadmaps/actions.ts

```diff
@@ -532,3 +532,15 @@ export const addTaskRelation = createAsyncThunk<
     return thunkAPI.rejectWithValue(err);
   }
 });
+
+export const removeTaskRelation = createAsyncThunk<
+  boolean,
+  TaskRelationRequest,
+  { rejectValue: AxiosError }
+>('removeTaskRelation', async (relationRequest, thunkAPI) => {
+  try {
+    return await api.removeTaskRelation(relationRequest);
```

Comment:
Changing the TaskRelationRequest to TaskRelation would simplify the task-map logic a bit
I have removed the type from addTaskRelation in another branch, my bad for implementing it this way in the first place
```suggestion
export const removeTaskRelation = createAsyncThunk<
  boolean,
  TaskRelation,
  { rejectValue: AxiosError }
>('removeTaskRelation', async (relationRequest, thunkAPI) => {
  try {
    const roadmapId = chosenRoadmapIdSelector(
      thunkAPI.getState() as RootState,
    )!;
    return await api.removeTaskRelation(roadmapId, relationRequest);
```

Reply:
Changed it to be like this 👍 Cleaned up the id's of the edges as well from `from-:taskId-to-:taskId-in-:roadmapId` to `from-:taskId-to-:taskId` since the roadmap gets selected in the actions